package/kedro_viz/data_access/managers.py

```diff
@@ -69,6 +69,21 @@ def set_db_session(self, db_session_class: sessionmaker):
         """Set db session on repositories that need it."""
         self.runs.set_db_session(db_session_class)
 
+    def resolve_dataset_factory_patterns(
+        self, catalog: DataCatalog, pipelines: Dict[str, KedroPipeline]
```

Comment:
```suggestion
        self, catalog: DataCatalog, pipelines: Dict[str, Pipeline]
```

Reply:
Hi Nok, we have an import with alias as - `from kedro.pipeline import Pipeline as KedroPipeline` . This is used in other places as well, so followed the same.